### ISSUE TITLE
How to check for video progress?

### ISSUE THREAD

**Issue by MirrorTM** [englercj/resource-loader#153]

**Comment by MirrorTM** [comment]
when loading a video i wish to wait until its fully loaded , however the loader doesnt report progress by onProgress.. how to manage this? cant find any documentation

**Comment by englercj** [comment]
I don't think we report individual resource load progress for anything that isn't XHR currently. PR welcome.

You'd need to implement it in here: https://github.com/englercj/resource-loader/blob/master/src/load_strategies/MediaElementLoadStrategy.ts
You can see how it works for XHR in here: https://github.com/englercj/resource-loader/blob/master/src/load_strategies/XhrLoadStrategy.ts
Here are the relevant MDN docs: https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/progress_event

**Comment by englercj** [comment]
Oh actually, we don't report anything after "canplaythrough" because we consider that "done". The best thing to do here is probably create your own video element and use that. You can even pass it into resource loader if you want, depending on what you're actually doing.
